tests/storage/export.go

```diff
@@ -1271,12 +1271,10 @@ var _ = SIGDescribe("Export", func() {
 		Expect(*snapshot.Status.SourceUID).To(Equal(vm.UID))
 
 		contentName := *snapshot.Status.VirtualMachineSnapshotContentName
-		Expect(snapshot.Status.Indications).To(BeEmpty())
```

Comment:
hmm whats wrong with these?

Reply:
If you are using image with guest agent, it should not be empty?  Do some tests have guest agent and some do not?  Maybe that can be a flag?

Reply:
But this aspect is not particularly relevant for this test suite

Reply:
We will keep pursuing this outside the PR